### ISSUE TITLE
Can't run make clean right after cloning the repository

### ISSUE THREAD

**Issue by arzam16** [arzam16/kernel_acer_b1_a71_xdavidwu#2]
```
scripts/Makefile.clean:17: /.../kernel_acer_b1_a71_xdavidwu/kernel/mediatek/custom/out/acer17_tb_wifi_only_jb/kernel/Makefile: No such file or directory
make[1]: *** No rule to make target '/.../kernel_acer_b1_a71_xdavidwu/kernel/mediatek/custom/out/acer17_tb_wifi_only_jb/kernel/Makefile'.  Stop.
Makefile:1232: recipe for target '_clean_mediatek/custom/out/acer17_tb_wifi_only_jb/kernel' failed
make: *** [_clean_mediatek/custom/out/acer17_tb_wifi_only_jb/kernel] Error 2
```

**Comment by arzam16** [comment]
The solution should be restoring the 'mediatek' directory from the original B1-710 source package.

**Comment by arzam16** [comment]
Fixed in 2c30c6f
